src/test/java/de/wejago/hichi2influx/service/SensorMessageSubscriberTest.java

```diff
@@ -29,15 +39,15 @@ public class SensorMessageSubscriberTest {
     @Test
     void messageArrived_withValidMessage_writesPointToInfluxDbRepository() throws Exception {
         // GIVEN
-        String receivedMessage = "{\"Time\":\"2023-02-12T21:29:36\",\"SML\":{\"1_8_0\":22417.98}}";
-        SensorEntry expectedSensorEntry = new SensorEntry();
+        String receivedMessage = "{\"Time\":\"2023-02-12T21:29:36\",\"SML\":{\"1_8_0\":22417.98}}".trim();
+        SensorEntry expectedSensorEntry = generateTestSensorEntry();
         when(objectMapper.readValue(receivedMessage, SensorEntry.class)).thenReturn(expectedSensorEntry);
 
         // WHEN
         subscriber.messageArrived("test", new MqttMessage(receivedMessage.getBytes()));
 
         // THEN
-        verify(influxDbRepository).writePoint(expectedSensorEntry);
+        verify(influxDbRepository, times(1)).writePoint(any());
```

Comment:
generateMeasurementPoint(SensorEntry sensorEntry) is a private method. If we want to check the content would be best to make it public, right?

Reply:
No, you just create a Point with the content you expect and then use that in the verify instead of any().
Or you use an ArgumentCaptor and assert the result is equal.